wiki/Client/Release_stream/Lazer/en.md

```diff
@@ -0,0 +1,43 @@
+---
+stub: true
+tags:
+  - client
+  - game client
+  - osu!lazer
+  - osu! lazer
+  - osu lazer
+  - osu!(lazer)
+---
+
+# Lazer release stream
+
+::: Infobox
+
+![](/img/lazer.png "The osu!(lazer) client logo, a stylised version of the osu! cookie")
+
+[Download](https://github.com/ppy/osu#running-osu) • [Issue tracker](https://github.com/ppy/osu/issues) • [Discussions](https://github.com/ppy/osu/discussions)
+
+|  |  |
+| :-- | :-- |
+| First release | 2017-02-12 |
+| Open source | Yes |
+
+:::
+
+*For common questions regarding updating to lazer, and the new client itself, see: [Help Centre/Upgrading to lazer](/wiki/Help_centre/Upgrading_to_lazer)*
+
+"Lazer" is the code name for the next major update to osu!. While the other release streams are feature-locked, lazer will continue to receive updates and eventually become the new "stable" client, with other versions renamed accordingly.
+
+## History
+
+The project started in mid-2015[^peppy-2017] as a new design for the stable client's interface codenamed "osu!next" made by [flyte](https://osu.ppy.sh/users/3103765), osu!'s lead designer. This was first showcased in a [teaser video](https://www.youtube.com/watch?v=lrWyqpNUwBo) the same year.
+
+Draft versions were uploaded on flyte's [design blog](https://osunext.tumblr.com/) starting from 2016.
+
+Implementing the design and new planned features using stable's code base proved to be difficult or even nearly impossible due to technical debt. As such, work started on a complete rewrite of the client early 2016 and a first release was published on GitHub on 12 February 2017, containing a bare-minimum implementation of gameplay in the osu! game mode, and the new menu designs.
+
+A framework is concurrently written as a separate project, the osu!framework, acting as a space for game components and systems that could be useful in other games.
```

Comment:
```suggestion
A framework is concurrently maintained as a separate project, the [osu!framework](https://github.com/ppy/osu-framework), to separate lower-level responsibilities at a code level. It is hoped that the framework could be useful in other games or applications.
```

Maybe also worth mentioning that it's already been used for some other projects? Out of scope? not sure.

Reply:
not sure. maybe not in the history section though